server/apps/archive/archive.py

```diff
@@ -143,6 +143,15 @@ class ArchiveService(BaseService):
     packageService = PackageService()
     mediaService = ArchiveMediaService()
 
+    def on_fetched(self, docs):
+        """
+        Overriding this to handle existing data in Mongo & Elastic
+        """
+
+        for item in docs[config.ITEMS]:
+            if 'pubstatus' in item:
+                item['pubstatus'] = item.get('pubstatus', 'Usable').capitalize()
```

Comment:
I will change the values as per IPTC Standards. However, this piece of code handles existing data not for display purpose.